sources/constraint-definition.asciidoc

```diff
@@ -436,7 +436,7 @@ If left undefined, the default value for [methodname]`@OverridesAttribute.name`
 
 [NOTE]
 ====
-[tck-testable]#A composing constraint can itself be a composed constraint. In this case, attribute values are overridden recursively according to the described rules.# Note however, that a forwarding rule (as defined by [classname]`@OverridesAttribute`) is only applied to the direct composing constraints.
```

Comment:
I understood it the other way around: that a meta annotation can also be used to compose other meta annotations. I think it's more logical to state it this way.